src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,30 +16,37 @@ import './WebexLocalMedia.scss';
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
-export default function WebexLocalMedia({meetingID}) {
+export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
 
-  const classBaseName = 'local-media';
-  const classObjects = {};
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-local-media`;
+  const mainClasses = {
+    [classBaseName]: true,
+    desktop: width >= PHONE_LARGE,
+    'no-media': localVideo === null,
+  };
```

Comment:
```suggestion
    className
  };
```

Reply:
Doing this generates the classes as:
`className="wxc-local-media wxc-local-media-no-media className"`

I found that adding `[className]: !!className,` works as expected though

Reply:
Interesting that it makes it a string, literally, instead of evaluating the value